packages/web-twig/src/Resources/components/Accordion/Accordion.twig

```diff
@@ -18,5 +18,5 @@
     {{ classProp(_classNames) }}
     data-toggle="accordion"
 >
-    {%- block content %}{% endblock %}
+    {%- block content %}{% endblock -%}
```

Comment:
Just to be sure, `-` characters remove whitespace in Twig templates (https://symfony.com/blog/better-white-space-control-in-twig-templates). In most places we have `{% block content %}{% endblock %}` or `{%- block content -%}{%- endblock -%}`. Are you sure about the usage in this case?

Reply:
Nope, this is what @adamkudrna suggested me

Reply:
Ok, thanks for the explanation.